about.html

```diff
@@ -0,0 +1,174 @@
+                        and connection.
+                    </p>
+                    <p class="d-none d-md-block white-bg m-5 p-4 ps-5 pe-5 text-wrap">
+                        Google Developer Groups represents a network of technical
+                        communities that offer <br />opportunities for professional
+                        development and connection. Those who join can learn about the
+                        latest Google<br /> tools and technologies, and can meet with 
+                        local developers with similar interests.
+                    </p>
+                    <div class="m-5">
+                        <img class="w-25 border border-2" src="img/gdg-member-badge.png" alt=""/>
+                    </div>
+                </div>
+            </div>
+            <div class="d-flex justify-content-center">
+                <div class="text-center">
+                    <div class="main-title position-relative m-5">
+                        <h2 class="fw-semibold">Earnable badges</h2>
+                    </div>
+                    <p class="white-bg mb-5 pb-4 ps-5 pe-5 text-wrap">
+                        Earn these badges when you visit and register for 2022 DevFest
+                        event.
+                    </p>
+                    <div class="d-flex d-md-none m-5 flex-column justify-content-center align-items-center">
+                        <img class="w-50 m-1 border border-2" src="img/badge-visit.svg" alt=""/>
+                        <img class="w-50 m-1 border border-2" src="img/badge-register.svg" alt=""/>
+                    </div>
+                    <div class="d-none d-md-flex m-5 justify-content-center">
+                        <img class="w-50 m-1 border border-2" src="img/badge-visit.svg" alt=""/>
+                        <img class="w-50 m-1 border border-2" src="img/badge-register.svg" alt=""/>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <footer id="footer" class="row d-md-flex pt-4 pb-4 ps-lg-5 ps-md-2">
+            <div class="ms-md-3 col-md-auto col-6 text-center">
+                <img id="logo" class="ps-2 pt-4 gdg" src="img/logo_dark_gdg.svg" alt=""/>
+            </div>
+            <div class="col-md-auto col-6 pt-lg-2 pt-md-2 ps-md-3 ps-lg-4 pe-3">
+                <h6 class="fontS-12 pt-3 pt-md-0">
+                    Google Developers under terms that explicitly encourage people to
+                    take, modify, reuse, re-purpose, and remix our work as they see fit.
+                </h6>
+                <h6 class="fontS-12 d-none d-md-block">
+                    Proper attribution is required when you reuse or create modified
+                    versions of content that appears on a page made available under the
+                    terms of the Creative Commons Attribution license.
+                </h6>
+            </div>
+        </footer>
```

Comment:
- [ ] As the design guidelines are, kindly ensure that the background color of the footer on desktop view is black.
- [ ] Also, kindly ensure that you center the content in your footer so that it can match the provided behance design and make it look more professional.

**This is how your about footer desktop view looks like:**

![image](https://user-images.githubusercontent.com/86318284/229049074-6663a694-0f53-4fae-b98c-d876e9218130.png)


**The desired design:**

![image](https://user-images.githubusercontent.com/86318284/229049118-80ec3373-d798-4076-b995-87aee696c913.png)


- [ ] On mobile view, kindly ensure that the content is centered and ensure that the font size that is used is inline with the provided one in the Behance design template. You can consider reducing the text size that is in the footer section to not distort the design.

**This is how your about footer mobile view looks like:**

![image](https://user-images.githubusercontent.com/86318284/229049702-a3dab24d-ef41-476b-a3f0-dc77b8d8ba99.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/229049788-3e89d3a0-c118-48b2-be25-f4c6b44e9dc5.png)

Reply:
@Nemwel-Boniface Thank you for your feedback.  Changes were submitted already.

Reply:
Well resolved :heavy_check_mark: